### PR TITLE
WTH Pull Request Review Process

### PR DESCRIPTION
**Pull request by jrzyshr**
Here's a V1 Markdown version of the WTH VTeam's Pull Request Review Process for Reviewers.

### CODE REVIEW

File: 000-HowToHack/WTH-PRReviewProcess.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    1.	If you choose “Approve”, this does NOT merge/publish the PR into the WTH repo.  This only indicates that as a reviewer, YOU “approve” the changes.  It takes TWO reviewers to APPROVE before the “Merge pull request” button lights up green.  Thus, if you are the first reviewer, you will not be able to merge the PR.  
+    ![PR With A Single Approval](./images/wth-pr-singleapproval.png?raw=true "PR With a Single Approval") 
+1.	Once you have completed your initial review, reach out to the second reviewer to make sure they know to complete their review.
+
+    If the second (or any other) reviewer has requested changes, it is the responsibility of the assigned owner of the PR to reach out to the author(s) to let them know it is their action to make the changes requested by the second (or any other) reviewer.
+
+    **NOTE:** If the PR has a lof of changes, it is a good practice for the assigned PR owner to coordinate with all reviewers to let them have a chance to review and request changes before reaching out to the author(s) to let the author(s) know they have an action.
```
Comment (Whowong) [inline 000-HowToHack/WTH-PRReviewProcess.md:103]: typo on "lot"

File: 000-HowToHack/WTH-PRReviewProcess.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    ![PR In-line Comments](./images/wth-pr-inlinecomments.png?raw=true "PR In-line Comments")
+    1. Start a review from the “Review Changes” button at the top of the “Files Changed” tab.  If you are not requesting any changes or just want to leave a single comment, you would start (and finish) the review this way.
+    ![PR Review Changes Button](./images/wth-pr-reviewchangesbutton.png?raw=true "PR Review Changes Button")
+1.	Do your review!  Follow the guidelines from the [WTH Author’s Guide](https://aka.ms/wthauthor) for help and be on the look out for the common issues:
+    1.	Folder Structure/File organization/Navigation links/etc
+    1.	Links from Student guide to the WTH repo
+    1.	Grammar/spelling/template consistency
+1.	When you are done making in-line comments or you if want to make a only single comment, click the “Review Changes” button at the top of the “Files Changed” tab.  You need to select one of the three options: Comment, Approve, or Request Changes.
```
Comment (Whowong) [inline 000-HowToHack/WTH-PRReviewProcess.md:93]: flipped some words in italics.

"or _if you_ want to make _only a_ single comment"

**Review by Whowong — CHANGES_REQUESTED** [review]

File: 000-HowToHack/WTH-PRReviewProcess.md (hunk 1 above)
Comment (jrzyshr) [reply 000-HowToHack/WTH-PRReviewProcess.md:103]: Done.

File: 000-HowToHack/WTH-PRReviewProcess.md (hunk 2 above)
Comment (jrzyshr) [reply 000-HowToHack/WTH-PRReviewProcess.md:93]: done.

**Review by Whowong — APPROVED** [review]

**Review by shbabylo — APPROVED** [review]
Reviewed content and approve.  Appreciate the breakout of responsibilities!

**Pull request merged**